src/appshell/view/dockwindow/dockwindow.cpp

```diff
@@ -69,11 +71,17 @@ DockWindow::DockWindow(QQuickItem* parent)
     m_statusbar->setSizeGripEnabled(false);
     m_window->setStatusBar(m_statusbar);
 
+    connect(m_window->menuBar(), &QMenuBar::triggered, this, [this](const QAction* action) {
+        emit actionTriggered(action->data().toString());
+    });
+
     WidgetStateStore::restoreGeometry(m_window);
 
     connect(m_pages.notifier(), &uicomponents::QmlListPropertyNotifier::appended, this, &DockWindow::onPageAppended);
     connect(this, &DockWindow::colorChanged, this, &DockWindow::updateStyle);
     connect(this, &DockWindow::borderColorChanged, this, &DockWindow::updateStyle);
+
+    shortcutsController()->init();
```

Comment:
Can we do this somewhere else? For example, in AppShell Module::onInit 
Better keep dock widgets independents, for eyas change its implementation (using an external solution)